src/Symfony/Component/PropertyInfo/PropertyInfoExtractor.php

```diff
@@ -90,6 +90,12 @@ public function isInitializable(string $class, string $property, array $context
     private function extract(iterable $extractors, string $method, array $arguments): mixed
     {
         foreach ($extractors as $extractor) {
+            if (!method_exists($extractor, $method)) {
+                trigger_deprecation('symfony/property-info', '7.1', 'Not implementing the "%s()" method in class "%s" is deprecated."', $method, $extractor::class);
+
+                continue;
```

Comment:
This does not look like the right solution to me as it means that a formerly used extractor will now no longer be used leading to a (potentially) different result.

Reply:
Also, I thought about strictly checking for `getType` and have it default to `getTypes` and it would work for return values that have only 1 Type in the return array of `getTypes`. I think this change was done before: https://github.com/symfony/symfony/pull/54694. Most of the code in property-info thinks BIGINT is string and not this dual type scenario.

Reply:
see #57459